qemu/tests/qemu_disk_img_convert.py

```diff
@@ -80,13 +81,20 @@ def run(test, params, env):
     if not md5:
         test.error("Fail to save tmp file")
     convert_test.destroy_vm()
-    n_params = convert_test.convert()
-    convert_test.compare_test(n_params)
-    convert_test.verify_info(n_params)
-    convert_test.start_vm(n_params)
 
-    # check md5sum after conversion
-    ret = convert_test.check_file(t_file, md5)
-    if not ret:
-        test.error("image content changed after convert")
-    convert_test.clean()
+    cluster_size_set = params.get("cluster_size_set")
```

Comment:
You could use  `cluster_size_set = params.get("cluster_size_set", "")`:
Please see virttest/utils_params:
```
class Params(IterableUserDict):

    """
    A dict-like object passed to every test.
    """
    lock = Lock()

    def __getitem__(self, key):
        """ overrides the error messages of missing params[$key] """
        try:
            return IterableUserDict.__getitem__(self, key)
        except KeyError:
            raise ParamNotFound("Mandatory parameter '%s' is missing. "
                                "Check your cfg files for typos/mistakes" %
                                key)

    def get(self, key, default=None):
        """ overrides the behavior to catch ParamNotFound error"""
        try:
            return self[key]
        except ParamNotFound:
            return default
```